modoboa/core/commands/templates/settings.py.tpl

```diff
@@ -297,6 +297,8 @@ SILENCED_SYSTEM_CHECKS = [
     "security.W019",  # modoboa uses iframes to display e-mails
 ]
 
+DISABLE_RSS = False
```

Comment:
I think this name is the best one to describe what you're implementing in this PR. Something like `DISABLE_DASHBOARD_EXTERNAL_QUERIES` would be better in my opinion.